test/client/fixtures/sandbox/node/document-test.js

```diff
@@ -547,3 +547,16 @@ test('an iframe should not contain self-removing scripts after document.close (G
 
     strictEqual(nativeMethods.querySelectorAll.call(document, '.' + SHADOW_UI_CLASSNAME.selfRemovingScript).length, 0);
 });
+
+test('querySelector should return element if selector contains href attribute with hash as value (GH-922)', function () {
```

Comment:
`querySelector should return an element if a selector contains the href attribute with hash as a  value (GH-922)`

by @MarinaRukavitsyna